master/getting-started/kubernetes/installation/gce.md

```diff
@@ -2,49 +2,21 @@
 title: Deploying Calico and Kubernetes on GCE
 ---
 
-These instructions allow you to set up a Kubernetes cluster with Calico networking on GCE using the [Calico CNI plugin][cni-plugin]. This guide does not setup TLS between Kubernetes components or on the Kubernetes API.
+There are several possibilities for [deploying Kubernetes on GCE][running-GCE-on-K8s]
+and with minimal adjustments Calico can be [installed](.) too.
+The information on this page provides the customization needed.
 
-## 1. Getting started with GCE
+### Setting up GCE networking
 
-These instructions describe how to set up two CoreOS Container Linux hosts on GCE.  For more general background, see
-[the CoreOS on GCE documentation][coreos-gce].
+Setting up Kubernetes will require [some GCE firewall rules][default-GCE-networking]
```

Comment:
In addition to the standard GCE firewall rules required by kubernetes, Calico will need a firewall rule to allow IP-in-IP traffic between hosts.